test/integration/notifications.test.js

```diff
@@ -494,7 +495,7 @@ describe('Integration: notifications', () => {
       });
     });
 
-    test('post prompt to re-subscribe after user loses access to repo', async () => {
+    test.only('post prompt to re-subscribe after user loses access to repo', async () => {
```

Comment:
rogue `only` -- this is probably causing the drop in test coverage?

Reply:
Whooops. Thanks for spotting that!